### PR TITLE
Cohort viz advanced

### PR DESCRIPTION
**Pull request by aconsilvio**
Adding more cohort vizs

### CODE REVIEW

**Comment by BrennaManning** [comment]
:strawberry:

File: public/javascripts/cohortVizController.js [hunk 1]
```diff
@@ -38,6 +40,16 @@ var cohortVizController = function($scope, $http, $state) {
...
+
+    var engagePeerList = [];
+    $scope.engagePeerData = formatPieData($scope.engagePeer, engagePeerList);
+
+    var engageAdultList = [];
+    $scope.engageAdultData = formatPieData($scope.engageAdult, engageAdultList);
+
+    var engageContentList = [];
+    $scope.engageContentData = formatPieData($scope.engageContent, engageContentList);
+
```
Comment (nmohamed) [inline public/javascripts/cohortVizController.js:52]: for all these, why not just do `$scope.engageContentData = formatPieData($scope.engageContent, []);` for each line

**Comment by nmohamed** [comment]
other than that,
:peach:

File: public/javascripts/cohortVizController.js (hunk 1 above)
Comment (aconsilvio) [inline public/javascripts/cohortVizController.js:52]: Yeah that is actually all done in the next pull request :)  I wasn't sure it it belonged here or there, so sorry about that

**Pull request closed without merging**